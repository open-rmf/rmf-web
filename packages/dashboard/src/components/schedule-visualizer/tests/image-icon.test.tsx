import { mount } from 'enzyme';
import ImageIcon from '../image-icon';
import React from 'react';

test('Renders correctly', () => {
  const container = mount(
    <svg>
      <ImageIcon iconPath={'test'} height={1} width={1} footprint={0.5} />
    </svg>,
  );
  expect(container).toMatchSnapshot();
});
