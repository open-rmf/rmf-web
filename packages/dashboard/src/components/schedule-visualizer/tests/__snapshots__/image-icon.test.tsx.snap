// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<svg>
  <Memo()
    footprint={0.5}
    height={1}
    iconPath="test"
    width={1}
  >
    <g
      transform="translate(-0.5 -0.5)"
    >
      <image
        height={1}
        href="test"
        onError={[Function]}
        width={1}
      />
    </g>
  </Memo()>
</svg>
`;
